CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+# Contributing
+
+## Running Shell Commands Locally 🐚
+
+To run test commands from within the repository, simply run your commands from the root of the repository and use `bin/rdme` instead of `rdme` so it properly points to the command executable, like so:
+
+```sh
+bin/rdme validate __tests__/__fixtures__/ref-oas/petstore.json
```

Comment:
super nit
```suggestion
./bin/rdme validate __tests__/__fixtures__/ref-oas/petstore.json
```

Reply:
`bin/rdme` works for me 🤔

Reply:
works for me as well, `./bin/rdme` is just alwyas how i've run it.